spec/commands/process_webhook_event_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe ProcessWebhookEvent do
+  def perform(*_args)
+    described_class.for(*_args)
+  end
+
+  context 'pull requests event' do
+    let!(:event) { 'pull_request' }
+    let!(:request) { {} }
+
+    before do
+      expect_any_instance_of(PullRequestService).to receive(:process).and_return(nil)
+    end
+
+    it 'call perform in pull request service' do
+      ProcessWebhookEvent.for(request: request, event: event)
```

Comment:
Podrías probar corriendo el test con `ProcessWebhookEvent.for(request: {}, event: event)`?

Reply:
no queda más leíble si coloco arriba lo que representa la variable?

Reply:
Mmm si el objetivo es que sea más leíble supongo que esta bien, creía que lo creabas porque no sabias que podías ponerlo hacerlo directo jaja.